docs/guides-agent-app.md

```diff
@@ -0,0 +1,433 @@
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-6.png">
+</p>
+
+3.  Under **Permissions set on this app** you should see that Voting app is also allowed to execute actions on behalf of the Agent app.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-7.png">
+</p>
+
+# Use cases
+
+## A. Voting in another organization
+
+### **1. Create another Democracy DAO**
+
+Exactly the same as before, revisit [step 0 of the Setup section](/docs/guides-agent-app.md) above if you need reminding 😋. 
+
+### **2. Mint a token to allow our first DAO (A) to vote in our new DAO (B)**
+
+We've now created two Democracy DAOs -- let's call them **A** and **B**. A has an Agent app, B doesn't. We want to allow A to vote in B.
+
+Remember that one needs to be a tokenholder of B to be able to vote in B. And that A's Agent app acts its external interface.  
+
+In other words, A's Agent app allows it to participate as a stakeholder in B.
+
+It follows that to allow A to vote in B we need to mint a token for A's Agent app in B. 
+
+To do this run:
+
+```
+dao exec <name of dao B> <token manager app address of dao B> mint <agent app address of dao A> 1000000000000000000 --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
+```
+
+Remember, you can find the addresses of the apps in any of your DAOs by running `dao apps <organization name> --environment aragon:rinkeby`.
+
+As you can see, we are using the `dao exec` command to interact with the `mint` method of B's [Token Manager](https://wiki.aragon.org/dev/apps/token-manager/) app.
+
+ [`mint`](https://wiki.aragon.org/dev/apps/token-manager/index.html#mint-tokens) is used to create new tokens and assign them to a receiver. It takes two arguments: a receiver address and the amount of tokens to be created.
+
+In our case the receiver is A's Agent App, and the amount of tokens to be created is 1.
+
+However, you should notice that instead of writing `1` as the second argument to `mint` we've gone with `1000000000000000000`.
+
+This is because a token has 18 decimals, so 1 unit of a token is actually 0.000000000000000001 tokens. This is not what we want.
+
+In order to mint a full token from the CLI we need to pass the full number, which will then be interpreted with 18 decimals.
+In our case this is a 1 followed by eighteen 0s, or `1000000000000000000`.
+
+Finally, the usual warning: running the above command will trigger a vote in B to create and send a token to A's Agent App: we'll need to vote *yes* to confirm the minting of the token.
+
+Do this either directly through the UI or by running:
+
+```
+dao exec <name of dao B> <voting app address of dao B> vote 0 true true --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
+```
+
+
+### **3. Create a vote in B to add a third entity**
```

Comment:
I suggest we explain that we will do that to test how to vote using the Agent app.

Reply:
I think the explanation around the new image at the end of step 3 addresses this.